tools/study-room-scraper/src/study-room-scraper.ts

```diff
@@ -0,0 +1,170 @@
+import type { StudyRoom, StudyLocation } from "@peterportal-api/types";
+import { load } from "cheerio";
+import fetch from "cross-fetch";
+import { studyLocations, getStudySpaces } from "libs/uc-irvine-lib/src/spaces";
+import * as winston from "winston";
+
+const ROOM_SPACE_URL = "https://spaces.lib.uci.edu/space";
+
+type StudyLocations = {
+  [id: string]: StudyLocation;
+};
```

Comment:
`Record` is a more optimal type for representing simple relations. Prefix with `Scraped` so its more obvious where it originates from. Personally, I don't think this type is necessary because the `Record` type is already succinct.

```suggestion
type ScrapedStudyLocations = Record<string, StudyLocation>;
```